Helper/PaymentResponseHandler.php

```diff
@@ -11,7 +11,10 @@
 
 namespace Adyen\Payment\Helper;
 
+use Adyen\Model\Checkout\CancelOrderRequest;
+use Adyen\Payment\Helper\Config as Config;
```

Comment:
import name is reduntant here
```suggestion
use Adyen\Payment\Helper\Config;
```